PaddleCV/image_classification/dist_train/README.md

```diff
@@ -108,3 +108,24 @@ The second figure shows speed-ups when using multiple GPUs according to the abov
 Speed-ups of Multiple-GPU Training of Resnet50 on Imagenet
 </p>
 
+## Deep Gradient Compression([arXiv:1712.01887](https://arxiv.org/abs/1712.01887)) for resnet
+
+#### Environment
+
+  - GPU: NVIDIA® Tesla® V100 
+  - Machine number * Card number: 4 * 4
```

Comment:
Just kindly remind that benchmark on 4 x 8 v100 or 8 x 8 v100 even 16 x 8 v100 should be added in the future.